### PR TITLE
Add ServiceMonitor for each service

### PR DESCRIPTION
**Pull request by jacobcrawford**
**Description of your changes:**

To get prometheus to scrape metrics we add a ServiceMonitor for each service of a Deployment

Checklist:

* [x] I have bumped the chart version
* [x] I have documented my changes if this is needed
* [x] I have described my changes in the "description of your changes" field
* [x] I have tested this change on a running cluster and the Argocd dashboard shows healthy and synced
* [ ] I have created the necessary tests in the chart, if they are possible/necessary
* [x] I have squashed commits if necessary

### CODE REVIEW

File: charts/tenant-base/chart/templates/servicemonitor.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{{- range $deploymentName, $val := .Values.deployments }}
+{{- if .enabled }}
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ printf "%s-%s" $.Release.Name $deploymentName }}
```
Comment (klausenbusk) [inline charts/tenant-base/chart/templates/servicemonitor.yaml:7]: I think we should add one or more labels here, to help identify which application the scraped metrics belongs to.

Comment (jacobcrawford) [reply charts/tenant-base/chart/templates/servicemonitor.yaml:7]: I have added the standard labels we use. Was that what you were thinking? 😄

**Review by MadsHT — APPROVED** [review]
LGTM

File: charts/tenant-base/chart/templates/servicemonitor.yaml (hunk 1 above)
Comment (klausenbusk) [reply charts/tenant-base/chart/templates/servicemonitor.yaml:7]: Hmm, are they added to the metrics? I don't see the labels in Grafana.

**Pull request merged**